DataFormats/Detectors/TPC/include/DataFormatsTPC/ClusterNativeHelper.h

```diff
@@ -76,6 +82,12 @@ struct ClusterNativeBuffer : public ClusterGroupHeader {
   value_type clusters[0];
 };
 
+// @struct ClusterNativeFlatHeader
+// Header for flat trivially-messageable transfer of linear cluster native array
+struct alignas(64) ClusterNativeFlatHeader {
```

Comment:
lets add a version number and members for `Constants::MAXSECTOR` and `Constants::MAXGLOBALPADROW`

Reply:
sure